src/panels/config/automation/condition/types/ha-automation-condition-time.ts

```diff
@@ -143,4 +182,41 @@ export class HaTimeCondition extends LitElement implements ConditionElement {
   private _valueChanged(ev: CustomEvent): void {
     handleChangeEvent(this, ev);
   }
+
+  private _dayValueChanged(ev: CustomEvent): void {
+    ev.stopPropagation();
+
+    const daySwitch = ev.currentTarget as WeekdayHaSwitch;
+
+    let days = this.condition.weekday || [];
+
+    if (daySwitch.checked) {
+      days.push(daySwitch.day);
+    } else {
+      days = days.filter((d) => d !== daySwitch.day);
+    }
+
+    days.sort((a: string, b: string) => DAYS[a] - DAYS[b]);
+
+    this.condition.weekday = days;
+
+    fireEvent(this, "value-changed", {
+      value: this.condition,
+    });
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      .weekday-toggle {
+        display: flex;
+        height: 40px;
```

Comment:
Why do we set the height to 40px? What is wrong with the default styling?

Reply:
It looks really bad without those 40px:

![1403c6bcebf56314e31d3a1dea595480](https://user-images.githubusercontent.com/1741838/94235206-7b810e80-ff0b-11ea-8a2e-fd4d13a80e07.gif)